pyproject.toml

```diff
@@ -1,3 +1,3 @@
 [tool.black]
 line-length = 100
-target-version = ['py37', 'py38']
+target-version = ['py38']  # as soon as higher version than 20.8b1 is released, we can move it to py39
```

Comment:
Should we say
`as soon as black > 20.8b1 is released and we start using it, we can move include py39 here` ?